resources/codebuild_projects.go

```diff
@@ -0,0 +1,782 @@
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "tags",
+				Description: "A list of tag key and value pairs associated with this build project",
+				Type:        schema.TypeJSON,
+				Resolver:    resolveCodebuildProjectsTags,
+			},
+			{
+				Name:        "timeout_in_minutes",
+				Description: "How long, in minutes, from 5 to 480 (8 hours), for CodeBuild to wait before timing out any related build that did not get marked as completed",
+				Type:        schema.TypeInt,
+			},
+			{
+				Name:        "vpc_config_security_group_ids",
+				Description: "A list of one or more security groups IDs in your Amazon VPC.",
+				Type:        schema.TypeStringArray,
+				Resolver:    schema.PathResolver("VpcConfig.SecurityGroupIds"),
+			},
+			{
+				Name:        "vpc_config_subnets",
+				Description: "A list of one or more subnet IDs in your Amazon VPC.",
+				Type:        schema.TypeStringArray,
+				Resolver:    schema.PathResolver("VpcConfig.Subnets"),
+			},
+			{
+				Name:        "vpc_config_vpc_id",
+				Description: "The ID of the Amazon VPC.",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("VpcConfig.VpcId"),
+			},
+			{
+				Name:        "webhook_branch_filter",
+				Description: "A regular expression used to determine which repository branches are built when a webhook is triggered",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("Webhook.BranchFilter"),
+			},
+			{
+				Name:        "webhook_build_type",
+				Description: "Specifies the type of build this webhook will trigger.",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("Webhook.BuildType"),
+			},
+			{
+				Name:        "webhook_filter_groups",
+				Description: "An array of arrays of WebhookFilter objects used to determine which webhooks are triggered",
+				Type:        schema.TypeJSON,
+				Resolver:    resolveCodebuildProjectsWebhookFilterGroups,
+			},
+			{
+				Name:        "webhook_last_modified_secret",
+				Description: "A timestamp that indicates the last time a repository's secret token was modified.",
+				Type:        schema.TypeTimestamp,
+				Resolver:    schema.PathResolver("Webhook.LastModifiedSecret"),
+			},
+			{
+				Name:        "webhook_payload_url",
+				Description: "The CodeBuild endpoint where webhook events are sent.",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("Webhook.PayloadUrl"),
+			},
+			{
+				Name:        "webhook_secret",
+				Description: "The secret token of the associated repository",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("Webhook.Secret"),
+			},
+			{
+				Name:        "webhook_url",
+				Description: "The URL to the webhook.",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("Webhook.Url"),
+			},
+		},
+		Relations: []*schema.Table{
+			{
+				Name:        "aws_codebuild_project_environment_variables",
```

Comment:
maybe we shouldn't save this since it might hold sensitive info, we can hash the values. @yevgenypats  taughts?

Reply:
One of the controls is related to this table.

Reply:
Ah fair enough we can keep it, I am thinking if we can just json it as well, since type is basically the value type correct? so there is no point to save that info in a jsonb in theory we can cast it in golang to a map[string]interface{} but I think type is redundant perhaps.

Reply:
done

Reply:
we need to check type in one of controls

Reply:
Why don't we just store a hash of the value? The control only checks the key, not the value